bridge-token-factory/tests/token_transfer.rs

```diff
@@ -63,161 +84,218 @@ impl From<Proof> for AugmentedProof {
     }
 }
 
-#[macro_export]
-macro_rules! call_json {
-    ($signer:expr, $contract:ident, $method:ident, $arg:tt, $gas:expr, $deposit:expr) => {
-        $signer.call(
-            $contract.clone(),
-            stringify!($method),
-            json!($arg).to_string().into_bytes().as_ref(),
-            $gas,
-            $deposit,
-        )
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:tt), $gas:expr, $deposit:expr) => {
-        call_json!($signer, $contract, $method, $arg, $gas, $deposit)
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:tt)) => {
-        call_json!(
-            $signer,
-            $contract,
-            $method,
-            $arg,
-            near_sdk_sim::DEFAULT_GAS,
-            near_sdk_sim::STORAGE_AMOUNT
-        )
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:tt), deposit=$deposit:expr) => {
-        call_json!(
-            $signer,
-            $contract,
-            $method,
-            $arg,
-            near_sdk_sim::DEFAULT_GAS,
-            $deposit
-        )
-    };
-}
-
-#[macro_export]
-macro_rules! call_borsh {
-    ($signer:expr, $contract:ident, $method:ident, $arg:expr, $gas:expr, $deposit:expr) => {
-        $signer.call(
-            $contract.clone(),
-            stringify!($method),
-            &$arg.try_to_vec().unwrap(),
-            $gas,
-            $deposit,
-        )
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:expr), $gas:expr, $deposit:expr) => {
-        call_borsh!($signer, $contract, $method, $arg, $gas, $deposit)
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:expr)) => {
-        call_borsh!(
-            $signer,
-            $contract,
-            $method,
-            $arg,
-            near_sdk_sim::DEFAULT_GAS,
-            near_sdk_sim::STORAGE_AMOUNT
-        )
-    };
-    ($signer:expr, $contract:ident.$method:ident($arg:expr), deposit=$deposit:expr) => {
-        call_borsh!(
-            $signer,
-            $contract,
-            $method,
-            $arg,
-            near_sdk_sim::DEFAULT_GAS,
-            $deposit
-        )
+fn err_is(result: &Result<CallExecutionDetails, anyhow::Error>, expected: &str) {
```

Comment:
let's rename it now to something more readable - e.g. `assert_error`

Reply:
fixed https://github.com/aurora-is-near/rainbow-token-connector/pull/159/commits/dbaa6d9c2228efae27e4d688eb0c09e61656bbf3